go/vt/sqlparser/sql.y

```diff
@@ -7704,6 +7708,7 @@ non_reserved_keyword:
 | JSON_VALID %prec FUNCTION_CALL_NON_KEYWORD
 | JSON_VALUE %prec FUNCTION_CALL_NON_KEYWORD
 | JSON_UNQUOTE %prec FUNCTION_CALL_NON_KEYWORD
+| POINT %prec FUNCTION_CALL_NON_KEYWORD
```

Comment:
Could you move this back to its original position. This list is alphabetically sorted.